google_guest_agent/oslogin.go

```diff
@@ -99,15 +99,18 @@ func (o *osloginMgr) set(ctx context.Context) error {
 		logger.Infof("Disabling OS Login")
 	}
 
-	if err := writeSSHConfig(enable, twofactor, skey); err != nil {
+	// Unstable configuration section is a section with no guarantee of long term compatibility/support.
```

Comment:
What does this comment mean?

Reply:
We don't have now a mechanism for feature flags or even development feature flags. This change is introducing a new mechanism for that, I'm making it more formalized with this [PR](https://github.com/GoogleCloudPlatform/guest-agent/pull/288#) - see the `type Unstable struct{}` part of it.

Reply:
I expanded the comments a bit further to make it clear. Since it's the first user of it I'll keep this comments in the same region of the code, I'll get it removed when we get the `cfg` package PR merged.